nmtwizard/framework.py

```diff
@@ -845,14 +843,13 @@ def build_vocab(
             )
             config["tokenization"] = preprocess_config
         elif isinstance(preprocess_config, list):
-            local_config["preprocess"] = utility.resolve_environment_variables(
-                preprocess_config
-            )
-            config["preprocess"] = preprocess_config
-            local_config["vocabulary"] = utility.resolve_environment_variables(
```

Comment:
I suggest to keep updating the `local_config` to make sure the 2 configurations remain synced.